MaterialDesignThemes.Wpf/PaletteHelper.cs

```diff
@@ -12,6 +13,30 @@ namespace MaterialDesignThemes.Wpf
 {
     public class PaletteHelper
     {
+        private static ResourceDictionary refinedBrushDictionary;
```

Comment:
In general setting static members from instance constructors is a bad idea. I would suggest making both of these readonly instance fields. This will also mean changing `GetBrush` to either be an instance method or have it take in a `ResourceDictionary` as a parameter.